common/src/main/resources/com/box/l10n/mojito/okapi/filters/okf_table_csv_mojito_adobe_magento.fprm

```diff
@@ -0,0 +1,9 @@
+#v1
+fieldDelimiter=,
+textQualifier="
+sourceIdColumns=1
+sourceColumns=1
+targetColumns=2
```

Comment:
Could you let me know why targetColumns is 2 and not 1?

Reply:
it is the index of the column where the target string will be. If you look at the `fr_FR.csv`, you can see the first column is the source (english) and the second column is the translation.

Reply:
ok that is the column index, makes sense, thanks